lr_face/experiments.py

```diff
@@ -35,48 +37,57 @@ def __str__(self):
         return '_'.join(map(str, [
             self.scorer,
             self.calibrator,
-            data_str,
+            # data_str,
```

Comment:
This was commented out to prevent super long plot names,  but I think we can remove the data_str altogether, also on line  35

Reply:
done